docs/reference/conventions.md

```diff
@@ -0,0 +1,93 @@
+---
+myst:
+  html_meta:
+    "description": "Development conventions, rules and best practices"
+    "property=og:description": "Development conventions, rules and best practices"
+    "property=og:title": "Development conventions, rules and best practices"
+    "keywords": "Plone, Seven, frontend, best practices, conventions, rules"
+---
+
+# Development conventions, rules and best practices
+
+This document explains the conventions, rules, and best practices we follow in the Plone 7 frontend codebase.
+We generally stick to the same models, artifacts, and conventions as in Volto, unless there's a good reason to change.
+For example, the blocks model structure stays the same since it's well established and widely used.
+Another example: in Volto, we use controlled forms everywhere.
+Even though React Router 7 introduced a new way to handle forms, we stick to controlled forms for consistency.
+Our widgets also follow almost the same interface as in Volto because controlled forms need controlled widgets.
+The configuration system remains based on `@plone/registry`, and we keep improving it.
+We follow the mantra: "if it works, don't change it."
+This document highlights the differences and additions compared to Volto conventions.
+
+Separate builds for public and CMSUI
```

Comment:
Is this supposed to be a heading or a sentence? If a heading, it needs markup and content.

Reply:
It was supposed that I elaborate that idea. I will.